libpod/container.go

```diff
@@ -416,6 +416,10 @@ func (c *Container) Init() (err error) {
 		return err
 	}
 
+	if c.state.State == ContainerStateCreated {
```

Comment:
Can you remove this and instead ignore ErrCtrExists when doing the Init() in start.go?